dashboard-app/src/dashboard/ByActivity/useAggregateDataByActivity.ts

```diff
@@ -0,0 +1,79 @@
+import { DependencyList, useEffect, useState } from 'react';
```

Comment:
Why is this showing up as a new file? Did you copy it instead of moving it?

Reply:
Nope, moved it.

Reply:
No you didn't:

<img width="183" alt="Screenshot 2019-06-24 at 11 15 21" src="https://user-images.githubusercontent.com/12719553/60011336-7a5e9300-9671-11e9-91fc-b23144f7522b.png">

Reply:
i blame git...